### PR TITLE
Tekton  based OADP non-admin example blog

### PR DESCRIPTION
**Pull request by weshayutin**
This provides OADP administrators an example of how to safely setup OADP with tekton to provide non-admin users the ability to backup and restore namespaces to which they have access.

### CODE REVIEW

**Comment by weshayutin** [comment]
@kaovilai @shubham-pampattiwar @sseago looking for a review to merge so the files can be in place in master for a blog that is now in draft. [1]

[1] https://developers.redhat.com/articles/2023/06/09/delegate-oadp-backup-and-restore-actions-safely-non-admin-users

**Comment by kaovilai** [comment]
404

**Comment by shubham-pampattiwar** [comment]
yeah same 404 cc: @weshayutin

File: blogs/tekton-oadp-nonadmin/oadp-tekton-container/get_restore_namespace.sh
Comment (sseago) [inline blogs/tekton-oadp-nonadmin/oadp-tekton-container/get_restore_namespace.sh:16]: If the restore calls for restoring to a different namespace, this may be the wrong check. Do we need to take namespace mappings into account, or are we explicitly excluding namespace mappings from support here? If the latter, do we verify that the restore created won't include namespace mappings?

File: blogs/tekton-oadp-nonadmin/README.md
Comment (weshayutin) [reply blogs/tekton-oadp-nonadmin/README.md:11]: thanks

File: blogs/tekton-oadp-nonadmin/oadp-tekton-container/get_restore_namespace.sh
Comment (sseago) [reply blogs/tekton-oadp-nonadmin/oadp-tekton-container/get_restore_namespace.sh:16]: @weshayutin if it's supported (even as advanced) then the above code is wrong, since it's checking permissions based on source namespace, not target namespace. Or would this check not run in the advanced case you're talking about?

**Comment by weshayutin** [comment]
@shubham-pampattiwar @kaovilai ya.. the blog is private until published.. not sure how to give you access, but I need this bit merged.

**Review by kaovilai — APPROVED** [review]
/hold for other reviewers

**Comment by weshayutin** [comment]
@sseago one more pass please :)

**Comment by kaovilai** [comment]
@weshayutin need hold or it's gonna auto merge with my lgtm
/hold

File: blogs/tekton-oadp-nonadmin/README.md
Comment (shubham-pampattiwar) [inline blogs/tekton-oadp-nonadmin/README.md:23]: NIT: crd's <-> cr's

File: blogs/tekton-oadp-nonadmin/README.md
Comment (shubham-pampattiwar) [inline blogs/tekton-oadp-nonadmin/README.md:42]: NIT: CRD <-> CR

File: blogs/tekton-oadp-nonadmin/README.md
Comment (shubham-pampattiwar) [inline blogs/tekton-oadp-nonadmin/README.md:97]: `buzz1` will also require **edit** access (edit is missing here)

**Review by shubham-pampattiwar — APPROVED** [review]
Cool stuff ! Couple of small nits, LGTM !

File: blogs/tekton-oadp-nonadmin/README.md
Comment (kaovilai) [reply blogs/tekton-oadp-nonadmin/README.md:23]: +1

Comment (weshayutin) [reply blogs/tekton-oadp-nonadmin/README.md:23]: danka

File: blogs/tekton-oadp-nonadmin/README.md
Comment (weshayutin) [reply blogs/tekton-oadp-nonadmin/README.md:42]: danka

**Review by sseago — APPROVED** [review]
lgtm

**Pull request merged**
